src/main/java/com/blamejared/crafttweaker/impl_native/world/ExpandServerWorld.java

```diff
@@ -0,0 +1,121 @@
+package com.blamejared.crafttweaker.impl_native.world;
+
+import org.openzen.zencode.java.ZenCodeType;
+
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.blamejared.crafttweaker_annotations.annotations.NativeTypeRegistration;
+
+import net.minecraft.util.SharedSeedRandom;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.server.ServerWorld;
+
+/**
+ * Represents the logical (server) implementation of the world. These are not
+ * limited to dedicated servers, they exist in single player worlds as part of
+ * the integrated server.
+ * 
+ * @docParam this world
```

Comment:
Since there is no way to actually make a world / server world, maybe have this as
```zenscript
world as MCServerWorld
```
Just so it is clearer for people reading it